index.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const fs = require('fs');
+//fileReader?
+const fileReader = require('./lib/file-reader.js')
+
+//TODO:from lib or data folders??
+
+fileReader(`${__dirname}/data/one.txt`, function(err, data){
+  if(err) throw err;
+  console.log('First Content:', data.toString('hex', 0, 8));
+});
+
+fileReader(`${__dirname}/data/two.txt`, function(err, data){
+  if (err) throw err;
+  console.log('Second Content:', data.toString('hex', 0, 8));
+});
+
+fileReader(`${__dirname}/data/three.txt`, function(err, data){
+  if (err) throw err;
+  console.log('Third Content:', data.toString('hex', 0, 8));
```

Comment:
Using data.string('hex', 0, 8) when you're invoking your fileReader function is redundant since you already have that logic in the fileReader function that you are requiring in from file-reader.js

If you're fileReader is working the way it is supposed to then you should be able to log the data. Also, I noticed in your test you nested fileReader's callbacks within each other- which is great- it allows you to read and log the files in order, however, you didn't do that here so the async is not accounted for in this case. Be sure to play around and refactor this file to work the same way it does in your test :)